src/frontend/src/lib/derived/balances.derived.ts

```diff
@@ -17,3 +18,7 @@ export const balanceZero: Readable<boolean> = derived(
 		nonNullish($balanceStore?.[$token.id]) &&
 		$balanceStore[$token.id]?.data.isZero() === true
 );
+
+export const anyBalanceNonZero: Readable<boolean> = derived([balancesStore], ([$balanceStore]) =>
```

Comment:
You need a global derived store? Isn't this use in a single place?

Reply:
I am using both in `Actions` (show/hide `Send`) and `ExchangeBalance` (change the string). If you want I can extract it and put it directly in the `Hero` and pass the prop (removing the derived)

Reply:
Already two, I guess it will be quickly three or four usages so makes sense, thanks for the explanation.